moveit_planners/trajopt/include/trajopt_interface/kinematic_terms.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
```

Comment:
This is not allowed, but I think it should be now: https://github.com/ros-planning/moveit/issues/1597

Reply:
Now its allowed :-)